client/me/purchases/payment-methods/payment-method-delete.tsx

```diff
@@ -59,7 +61,7 @@ const PaymentMethodDelete: FunctionComponent< Props > = ( { card } ) => {
 				scary
 				borderless
 			>
-				{ text }
+				{ isNarrowView ? <Gridicon icon="trash" /> : text }
```

Comment:
This is optional. I thought it might look nicer in narrow viewports, but it doesn't directly address any issues reported in https://github.com/Automattic/wp-calypso/issues/93710

Reply:
In Checkout, we've intentionally moved away from using icons because of their ambiguity, and I think we intentionally expanded the text here in #62576, so that it was explicit that the entire payment method was being deleted (instead of just the billing information). Since the action is on its own line, I don't know that we're restricted by space.

Reply:
Makes sense. Thanks for the quick feedback. Happy to drop it as it was a cheeky proposal and, as mentioned, doesn't speak to the issue.